subiquity/common/types.py

```diff
@@ -349,6 +366,9 @@ def supports_manual_customization(self) -> bool:
                         GuidedCapability.LVM,
                         GuidedCapability.LVM_LUKS]
 
+    def is_zfs(self) -> bool:
```

Comment:
I guess this method is worth including because there will be GuidedCapability.ZFS_ENCRYPTED at some point?

Reply:
Yes, I thought it best to add this method now, as I do want to handle the ENCRYPTED case this cycle.